static/app/components/events/interfaces/crashContent/exception/sourceMapDebug.spec.tsx

```diff
@@ -14,7 +14,7 @@ import {
 jest.mock('sentry/utils/analytics/trackAdvancedAnalyticsEvent');
 
 describe('SourceMapDebug', () => {
-  const organization = TestStubs.Organization({features: ['fix-source-map-cta']});
```

Comment:
Are we going to delete this feature flag?

Reply:
@evanpurkhiser yes, needs to happen after. working on that PR now.